### PR TITLE
pointer events - add js feature to css property page

### PR DESCRIPTION
**Pull request by BobSquarePants**
### Description

Add the JS name to access this object from JS.

### Motivation

CSS name and JS name are not the same :/

### Additional details
NA

### Related issues and pull requests
NA

### CODE REVIEW

File: files/en-us/web/css/pointer-events/index.md [hunk 1]
```diff
@@ -80,6 +80,10 @@ Elements with `pointer-events: none` will still receive focus through sequential
 
 {{csssyntax}}
 
+## Javascript name
```
Comment (estelle) [inline files/en-us/web/css/pointer-events/index.md:83]: We don't include "JavaScript name" sections on CSS page, but we do link to related content under the "See Also" section at the bottom of (almost) every CSS property page.

File: files/en-us/web/css/pointer-events/index.md [hunk 2]
```diff
@@ -80,6 +80,10 @@ Elements with `pointer-events: none` will still receive focus through sequential
...
+
+`pointerEvents`
```
Comment (estelle) [inline files/en-us/web/css/pointer-events/index.md:85]: When referencing an API, or really any feature, link to it. 

When cross-referencing, you can use a [link macro](https://developer.mozilla.org/en-US/docs/MDN/Writing_guidelines/Page_structures/Links#cross-reference_links).  See `domxref` macro. For example, if you want to link to https://developer.mozilla.org/en-US/docs/Web/API/PointerEvent, you would use `{{domxref("PointerEvent")}}`.

**Review by estelle — COMMENTED** [review]
Hi-

Thank you for the suggestion. Including the PointerEvents API on this page is a good idea. The solution to this is to include a link, not just the name. The best place to include such links is in the see also section of the page.

**Comment by BobSquarePants** [comment]
@estelle I've made some change.
I hope it reflect better the MDN way.. ? (first contribution to MDN :) )
Cheers

File: files/en-us/web/css/pointer-events/index.md [hunk 3]
```diff
@@ -134,5 +130,6 @@ a[href="http://example.com"]
 
 - The SVG attribute {{SVGAttr("pointer-events")}}
 - The SVG attribute {{SVGAttr("visibility")}}
+- The PointerEvent API {{domxref("PointerEvent")}}
```
Comment (estelle) [inline files/en-us/web/css/pointer-events/index.md:133]:
```suggestion
- The {{domxref("PointerEvent")}} API
```

**Review by estelle — APPROVED** [review]
Thank you. Also, congratulations on your first merged MDN content PR. Welcome to the team!

**Pull request merged**